pkgs/applications/networking/instant-messengers/nheko/default.nix

```diff
@@ -48,23 +49,25 @@ mkDerivation rec {
   #  export CFLAGS=-Wno-error=gnu-zero-variadic-macro-arguments
   #'';
 
-  postPatch = ''
-    mkdir -p .deps/include/
-    ln -s ${tweeny}/include .deps/include/tweeny
-    ln -s ${spdlog} .deps/spdlog
-  '';
-
-  cmakeFlags = [
-    "-DTWEENY_INCLUDE_DIR=.deps/include"
-    "-DLMDBXX_INCLUDE_DIR=${lmdbxx}"
-    "-Dnlohmann_json_DIR=${nlohmann_json}/lib/cmake/nlohmann_json"
+  nativeBuildInputs = [
+    lmdbxx
+    cmake
+    pkgconfig
   ];
 
-  nativeBuildInputs = [ cmake pkgconfig ];
-
   buildInputs = [
-    mtxclient olm boost lmdb spdlog cmark
-    qtbase qtmultimedia qttools
+    nlohmann_json
+    tweeny
+    mtxclient
+    olm
+    boost17x
+    lmdb
+    spdlog
+    cmark
+    qtbase
+    qtmultimedia
+    qttools
+    qtquickcontrols2
```

Comment:
Have you tested this? I think you'll have to add qtgraphicaleffects to the inputs too

Reply:
I did manage to make it launch and login but I had nothing showing up in conversations and I assumed it's an upstream bug. Now with `qtgraphicaleffects` it's added to the environment of the wrapper and the conversations show themselves so :+1:. Thanks.